src/zabapgit_dot_abapgit.prog.abap

```diff
@@ -103,7 +108,7 @@ CLASS lcl_dot_abapgit IMPLEMENTATION.
     DATA: ls_data TYPE ty_dot_abapgit.
 
 
-    ls_data-master_language = iv_master_language.
+    ls_data-master_language = sy-langu.
```

Comment:
there is a check in deserialize, where the language is checked against the remote .abapgit.xml file, so it should work anyhow,